plugins/com.google.cloud.tools.eclipse.login.test/src/com/google/cloud/tools/eclipse/login/ui/AccountSelectorTest.java

```diff
@@ -388,14 +314,35 @@ public void testGetAccountCount_threeAccounts() {
   @Test
   public void testInitialItemOrder() {
     when(loginService.getAccounts())
-        .thenReturn(new LinkedHashSet<>(Arrays.asList(account3, account2, account1)));
+        .thenReturn(new HashSet<>(Arrays.asList(account3, account2, account1)));
```

Comment:
Yeah, I get that, but it's trouble to use it considering https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1657#discussion_r107978531.